astropy/io/fits/file.py

```diff
@@ -49,6 +50,9 @@
     'wb': 'ostream', 'wb+': 'update',
     'ab': 'ostream', 'ab+': 'append'}
 
+# A match indicates the file was opened in text mode, which is not allowed
+TEXT_RE = re.compile('^[rwa]((t?\+?)|(\+?t?))$')
```

Comment:
Okay, I just wanted to check since the development workflow says to rebase [only if asked](http://docs.astropy.org/en/stable/development/workflow/development_workflow.html#rebase-but-only-if-asked) :smile: